### ISSUE TITLE
Feedstocks that pin pytorch-cpu

### ISSUE THREAD

**Issue by h-vetinari** [conda-forge/pytorch-cpu-feedstock#155]
We should clean these up, because they're preventing users from picking CUDA-enabled pytorch for no good reason:

* [ ] [eeyore](https://github.com/conda-forge/eeyore-feedstock/blob/main/recipe/meta.yaml#L25)
* [ ] [htm-pytorch](https://github.com/conda-forge/htm-pytorch-feedstock/blob/main/recipe/meta.yaml#25)
* [ ] [nlpaug](https://github.com/conda-forge/nlpaug-feedstock/blob/main/recipe/meta.yaml#L27)
* [ ] [sagemaker-pytorch-serving-container](https://github.com/conda-forge/sagemaker-pytorch-serving-container-feedstock/blob/main/recipe/meta.yaml#L30)
* [ ] [sparktorch](https://github.com/conda-forge/sparktorch-feedstock/blob/main/recipe/meta.yaml#23)
* [ ] [torchbiggraph](https://github.com/conda-forge/torchbiggraph-feedstock/blob/main/recipe/meta.yaml#L35)
* [ ] [yews](https://github.com/conda-forge/yews-feedstock/blob/main/recipe/meta.yaml#L28)

Found by this [search](https://github.com/search?q=org%3Aconda-forge+-+pytorch-cpu&type=code)

**Comment by h-vetinari** [comment]
Thanks @mgorny for tackling the remaining feedstocks here! 🙏

**Comment by hmaarrfk** [comment]
all checked off!!! Time to close?

**Comment by h-vetinari** [comment]
> Found by this [search](https://github.com/search?q=org%3Aconda-forge+%2F-+pytorch-%5Bcg%5Dpu%2F+path%3Arecipe%2F*.yaml&type=code)

New ones keep appearing 😑

**Comment by hmaarrfk** [comment]
many are test time dependencies to try to stop (large) gpu packages from getting pulled in. we are ok with that right?

**Comment by h-vetinari** [comment]
Yeah, test dependencies are ok of course. It's only about avoiding to force a choice of GPU or non-GPU on ambient packages, so stuff in `run:` basically

**Comment by hmaarrfk** [comment]
i can't find any from the search. is your point that you often go back to the search and check up on this issue?

**Comment by h-vetinari** [comment]
"often" is not the word I'd use. Maybe once or twice a year? Last time was in February, which yielded one new issue:
* https://github.com/conda-forge/isofit-feedstock/issues/56
